### PR TITLE
feat(connect): add arns connect wallet modal

### PR DESCRIPTION
**Pull request by atticusofsparta**
image
image

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { Dispatch } from 'react';
```
Comment (dtfiedler) [inline src/types.ts:1]: nit: needed?

Comment (atticusofsparta) [reply src/types.ts:1]: nope, removed

File: src/components/icons/index.ts [hunk 2]
```diff
@@ -1,3 +1,7 @@
 import { ReactComponent as SearchIcon } from './Search.svg';
+import ArConnectIcon from './ArConnectIcon.png';
+import ArweaveAppIcon from './ArweaveAppIcon.png';
+import UploadJsonKey from './UploadJsonKey.png';
+import { ReactComponent as CloseIcon } from './Close.svg';
 
-export { SearchIcon };
+export { SearchIcon, CloseIcon, ArConnectIcon, ArweaveAppIcon, UploadJsonKey };
```
Comment (dtfiedler) [inline src/components/icons/index.ts:7]: super nit: alphabatize

File: src/components/modals/ConnectWalletModal/ConnectWalletModal.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <button className="walletConnectButton">
+          <img src={ArConnectIcon} alt="" width="47px" height="47px" />
+          Connect via ArConnect
+        </button>
+        <button className="walletConnectButton">
+          <img src={ArweaveAppIcon} alt="" width="47px" height="47px" />
+          Connect using Arweave.app
+        </button>
+        <text className="textFaded" style={{ marginTop: '1em' }}>
+          Don't have a wallet? get one{' '}
```
Comment (dtfiedler) [inline src/components/modals/ConnectWalletModal/ConnectWalletModal.tsx:53]: it's a bit more clear to use `&nbsp;` then the bracketed space

File: src/components/modals/ConnectWalletModal/styles.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  box-shadow: 0px 0px 4px rgba(255, 255, 255, 0.25);
+  border-radius: 24px;
+  padding: 30px;
+  justify-content: center;
+  align-items: center;
+  row-gap: 1em;
+}
+.walletConnectButton {
+  display: flex;
+  width: 100%;
```
Comment (dtfiedler) [inline src/components/modals/ConnectWalletModal/styles.css:62]: nit: you don't need to set `width:100%` in flexbox with no other elements

**Review by dtfiedler — APPROVED** [review]
some non-obstructive nit picky things, otherwise looks good

File: src/components/modals/ConnectWalletModal/ConnectWalletModal.tsx (hunk 3 above)
Comment (atticusofsparta) [reply src/components/modals/ConnectWalletModal/ConnectWalletModal.tsx:53]: this is actually from prettier - is their a config for this?

File: src/components/modals/ConnectWalletModal/styles.css (hunk 4 above)
Comment (atticusofsparta) [reply src/components/modals/ConnectWalletModal/styles.css:62]: Apparently you do? I think the image/text inside counts as an element, just defaults to fit-content.

**Review by dtfiedler — APPROVED** [review]

File: src/components/modals/ConnectWalletModal/ConnectWalletModal.tsx (hunk 3 above)
Comment (dtfiedler) [reply src/components/modals/ConnectWalletModal/ConnectWalletModal.tsx:53]: i think it'll add it if we don't explicitly add our own. also nit, these can be `spans` - not sure the difference with `text` tags

**Review by dtfiedler — APPROVED** [review]

**Pull request merged**